e2e/suite1/specs/select-focus.spec.ts

```diff
@@ -17,15 +17,15 @@ e2e.scenario({
 
         e2e.components.Select.option().should('be.visible').first().click();
 
-        e2e.components.Select.input().should('be.visible').should('have.focus');
+        e2e.components.Select.input().should('have.focus');
```

Comment:
Maybe we should add back the exist?
```suggestion
        e2e.components.Select.input().should('exist').should('have.focus');
```

Reply:
Updated the PR description for why this needs to be removed for v6 of cypress 😄